### PR TITLE
Translations/: set Zzzz as SleepingSimpleString in all translations for unification

### PR DESCRIPTION
**Pull request by ia**
* **Please check if the PR fulfills these requirements**
- [x] The changes have been tested locally
- [x] There are no breaking changes

* **What kind of change does this PR introduce?**
Translation update.

* **What is the current behavior?**
`SleepingSimpleString` is different in many translation files and sometimes is above 4 chars.

* **What is the new behavior (if this is a feature change)?**
`SleepingSimpleString` is set as `Zzzz` for all translations for unification.

* **Other information**:
Basically, this PR is inspired by [this commit](https://github.com/Ralim/IronOS/pull/1864). My [suggestion there](https://github.com/Ralim/IronOS/pull/1864#issuecomment-1873465419) was just to short _"localized"_ "mimic" sound(s) to fit, but I guess more appropriate approach is to make it more unified. Let me know what you think.

### CODE REVIEW

**Comment by discip** [comment]
@ai
As you should have noticed, in most languages ​​this expression was represented by **`Zzz`** rather than **`Zzzz`**.
This approach was chosen due to the fact that there should be a gap between the **`Zzz`** and the temperature reading, even if the value **exceeds 99**.

I think we should stick with this choice. 😊

Other than that this can be merged.

**Comment by ia** [comment]
> @ai As you should have noticed, in most languages ​​this expression was represented by **`Zzz`** rather than **`Zzzz`**. This approach was chosen due to the fact that there should be a gap between the **`Zzz`** and the temperature reading, even if the value **exceeds 99**.

Ahaaa... I see, got it! Thanks for this input. Well, it means that unification makes even more sense then since it's not easy to put _localized_ snoring sound from some other languages to 3 chars only.

> I think we should stick with this choice. 😊

Sure, no problem. I will update patch in a little bit.

**Comment by ia** [comment]
Done.

Plus, I updated `maxLen` for `SleepingSimpleString` in `Translations/translations_definitions.json` since it should be `3` to reflect the actual limit.

**Review by discip — APPROVED** [review]

**Comment by discip** [comment]
@ia
Unfortunately I was not precise about what term exactly needs to be used.
Turns out it should read: **`Zzz `** please note the space after the last **`z`**.
I really don't know how to handle this best.

I've never reverted a PR before, so let's see how this goes.

**Comment by discip** [comment]
@ia
Unfortunately I was not precise about what term exactly needs to be used.
Turns out it should read: **`Zzz `** please note the space after the last **`z`**.
Meaning the total number of chars remains 4.
I really don't know how to handle this best.

I've never reverted a PR before, so let's see how this goes.

**Comment by discip** [comment]
@ia
Unfortunately I was not precise about what term exactly needs to be used.
Turns out it should read: **`Zzz `** please note the space after the last **`z`**.
Otherwise the temperature value will be shifted 1 char to the left remaining glued to the **`z`**. 🤦🏻
Meaning the total number of chars remains **4**.

**Comment by discip** [comment]
@ia
Unfortunately I was not precise about what term exactly needs to be used.
Turns out it should read: **`Zzz `** please note the space after the last **`z`**.
Otherwise the temperature value will be shifted 1 char to the left remaining glued to the **`z`**. 🤦🏻
Meaning the total number of chars also remains **4**.

**Comment by ia** [comment]
> @ia Unfortunately I was not precise about what term exactly needs to be used. Turns out it should read: **`Zzz `** please note the space after the last **`z`**. Otherwise the temperature value will be shifted 1 char to the left remaining glued to the **`z`**. 🤦🏻 Meaning the total number of chars also remains **4**.

Ahaaaaaa...aye see! How interesting. Thanks for pointing this out, @discip. Sure, no problem! I will do a couple of tests & checks first but then I will provide updated patch to that.

Sorry for my misunderstanding & lack of testing. Although the lesson which I should learn by now: the more trivial potential change looks - the bigger _"undersea rocks"_ are somewhere out there! ;)
